### ISSUE TITLE
Some Characters are Different when using Arduino Uno and Nano Every

### ISSUE THREAD

**Issue by leonka10** [Andy4495/NewhavenOLED#3]
Hello, i hope u can help me. When i try the example code on Arduino Uno everything works fine. When I use the exact same Code and Wiring on Arduino Nano Every it works but some Characters are displayed as different Characters. For Example W becomes U and O becomes M. I don't know why and how to fix this.

**Comment by Andy4495** [comment]
The higher clock speed of the Every may be causing a timing issue. I don't have an Every to test with, but have some other faster processors that I can try some things. I'll look at the data sheets and confirm the timing specs and see where there might be a problem.

**Comment by Andy4495** [comment]
I was unable to reproduce the problem, even when testing with a 120 MHz processor. However, in reviewing the controller chip spec, the library was not quite matching the signaling as described in the spec. I made updates to both the library and example code on the `issue_3` branch. I have tested the updated code on an UNO, a 25 MHz MSP430, and a 120 MHz Tiva processor and all appear to work. If you have time, replace your library files with the the attached, and see if that fixes the problem.

[NewhavenOLED-issue_3.zip](https://github.com/user-attachments/files/16103159/NewhavenOLED-issue_3.zip)

You can replace the whole library or just the specific files that changed: 
```text
./library.properties
src/NewhavenOLED.cpp
src/NewhavenOLED.h
examples/Newhaven_OLED_example/Newhaven_OLED_example.ino
```

**Comment by Andy4495** [comment]
@leonka10: I have published a new version of the library [v1.1.4](https://github.com/Andy4495/NewhavenOLED/releases/tag/v1.1.4). 

If you have time, please check and see if it fixes your issue. If you continue to have problems, open a new issue and include the sketch and a hookup diagram/photo and I can try to look into it further.
